scripts/Muon_Analysis_2.py

```diff
@@ -11,21 +11,82 @@
 
 import PyQt4.QtGui as QtGui
 import PyQt4.QtCore as QtCore
-
+from mantid.kernel import ConfigServiceImpl
+from Muon.GUI.Common.muon_data_context import MuonDataContext
 from Muon.GUI.Common.dummy_label.dummy_label_widget import DummyLabelWidget
 from Muon.GUI.MuonAnalysis.dock.dock_widget import DockWidget
 from Muon.GUI.Common.muon_context.muon_context import *  # MuonContext
 from save_python import getWidgetIfOpen
+from Muon.GUI.MuonAnalysis.load_widget.load_widget import LoadWidget
+import Muon.GUI.Common.message_box as message_box
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+
 
 Name = "Muon_Analysis_2"
 
+
 muonGUI = None
+SUPPORTED_FACILITIES = ["ISIS", "SmuS"]
```

Comment:
what is SmuS?

Reply:
I assumed it was some other Muon facility, if you don't know what it is shall I just remove it?

Reply:
see https://muonsources.org/muon-centres.html at the moment we only support ISIS but we can in theory support PSI as we have a data loader for it now